tests/pages/components/github-issue-select.js

```diff
@@ -0,0 +1,37 @@
+import { attribute, findElementWithAssert, collection } from 'ember-cli-page-object';
+import { clickTrigger, nativeMouseUp } from 'code-corps-ember/tests/helpers/ember-power-select';
+
+export default {
+  scope: '.github-issue-select',
+
+  openDropdown() {
```

Comment:
Does it potentially make sense to utilize the `pages/components/power-select` file and maybe make it more dynamic (ie. pass `.github-issue-select`)?  Looks like somehow the `trigger.open` in the PS page file could take `this.scope` from this file?